### ISSUE TITLE
Add BG change and time of last reading

### ISSUE THREAD

**Issue by tdawggcat** [jpegjames/glukey#4]
Awesome app!  Thanks for supporting the community!  I help support many users in the CGM In The Cloud group on FaceBook.  Many people use the old, cruddy Nightscout Menubar program. I could switch several users over to GluKey, but they've told me they rely on the change/time.

So, the current reading I see on my bar is "237 -> -7 [6m]".

Just listing here as a feature request as you consider future builds.  Thanks so much!

**Comment by jpegjames** [comment]
@tdawggcat: Thanks for the feedback. I will look into adding this to as an option for a future release. My plan for the next major tweak is to add support for Nightscout API endpoints.  

I don't know when I will be able to get to this one, but it shouldn't be difficult to add.

**Comment by tdawggcat** [comment]
Yeah, I saw your reference to Nightscout when I was perusing the code.  Very nicely done.  I only code in scripting really, packing things together like this is beyond my skill set, otherwise I'd offer to help out.  I'm still perusing though.  I'm really impressed with what you've done.

**Comment by jpegjames** [comment]
Thanks! I try to document my code in a lot of detail…not only for others who might work on it, but for me since this is a side project and there can be long stretches in-between working on it.

This is actually my first and only Swift app, so I know there's still a lot of room for improvement. I'm normally a Ruby developer working primarily on web apps.

**Comment by tdawggcat** [comment]
I've got a couple of ideas for some apps that I've been talking to some of
my developers about (I run an IT company). I still haven't gotten around to
doing anything about them.  They'd take a little more time, so might need
to be ad supported, not sure.  Shoot me an email if you ever want to
brainstorm.  treythompson@gmail.com

FYI, one of the apps I've been thinking about is a QuickEntry app for
Nightscout.  I built a page where my daughter can quickly enter things into
her NS site, and I'd like to make it a mobile app so that it's got more
capabilities, offline entry, etc...  Anyway, thanks again for the great,
selfless work!
